addons/web/static/src/views/fields/properties/properties_field.scss

```diff
@@ -115,6 +104,21 @@
     .oi-draggable {
         cursor: url(/web/static/img/openhand.cur), grab;
     }
+    // mobile view
+    @include media-breakpoint-down(sm) {
+        .o_field_property_open_popover,
+        .oi-draggable {
+            margin-left: .75em !important;
+            margin-right: .75em;
+            opacity: 1;
+        }
+        .o_field_property_add {
+            display: none;
+        }
```

Comment:
can this be bootstrap classes ? (it wasn't before because we needed o_xxs_form_view)

Reply:
Hi @std-odoo :slightly_smiling_face: 

There is no responsive alternative for the opacity so I have to keep it in the css :confused: 
I guess the display none and the margins could go in the xml indeed. I don't think this is an issue. :thinking: 

In the master fw-port, I'll remove the unnecessary div and set the \<style> tag back in the css :slightly_smiling_face: 

Thanks and cheers!